core/assets/bundles/bundle_nl.properties

```diff
@@ -64,51 +64,51 @@ customgame = Aangepast spel
 newgame = Nieuw spel
 none = <none>
 minimap = Minimap
-position = Position
-close = Aflsuiten
+position = Positie
+close = Afsluiten
 website = Website
 quit = Stoppen
-save.quit = Save & Quit
+save.quit = Opslaan & Stoppen
 maps = Mappen
-maps.browse = Browse Maps
+maps.browse = Door mappen bladeren
 continue = Ga door
 maps.none = [LIGHT_GRAY]Geen map gevonden!
-invalid = Invalid
-preparingconfig = Preparing Config
-preparingcontent = Preparing Content
-uploadingcontent = Uploading Content
+invalid = Ongeldig
+preparingconfig = Configuratie voorbereiden
+preparingcontent = Inhoud voorbereiden
+uploadingcontent = Inhoud uploaden
 uploadingpreviewfile = Uploading Preview File
 committingchanges = Comitting Changes
-done = Done
-feature.unsupported = Your device does not support this feature.
-mods.alphainfo = Keep in mind that mods are in alpha, and[scarlet] may be very buggy[].\nReport any issues you find to the Mindustry GitHub or Discord.
+done = Klaar
+feature.unsupported = Uw apparaat ondersteunt deze toepassing niet.
```

Comment:
"feature" should not be translated with "toepassing" in this context. Feature in this context refers to a specific thing the application (the game) can't do on a device. A suggestion: "functie" (From nl_BE file.)